lib/index.js

```diff
@@ -24,19 +33,47 @@ document.querySelector(DOMstrings.searchButton).addEventListener('click', functi
     success: function(data) {
       const tracks = data.message.body.track_list
       const artistHtml = `<h3>${artist.toUpperCase()}</h3>`
+      const trackHtml = `<li>%trackName% <img src="./empty-star.png" alt="favorite" class="favorite-star"></li>`
       document.querySelector(DOMstrings.searchedArtist).insertAdjacentHTML('beforeend', artistHtml)
-      tracks.forEach(track => {
-        const trackHtml = `<li>${track.track.track_name} <img src="./empty-star.png" alt="favorite" class="favorite-star"></li>`
-        document.querySelector(DOMstrings.searchResults).insertAdjacentHTML('beforeend', trackHtml)
+      makeTrackArray(tracks, artist);
+      trackObjArray.forEach(track => {
+        let newTrackHtml = trackHtml.replace('%trackName%', track.trackName)
+        document.querySelector(DOMstrings.searchResults).insertAdjacentHTML('beforeend', newTrackHtml)
       })
     }
   })
   resetSearchField();
 });
 
+function makeTrackArray(trackData, artistData) {
+  trackData.forEach(track => {
```

Comment:
This could be reformatted slightly as `return trackData.map(...etc` and then you wouldn't have to define `trackObjArray` separately. It would be a bit cleaner but I don't think it's a big deal either way